### ISSUE TITLE
accessibility issues: new file drag and drop contrast too low?

### ISSUE THREAD

**Issue by shiffman** [processing/p5.js-web-editor#709]
In the "high contrast mode" on Mac OS (chrome browser) the instructions do not pass WCAG contrast tests:

> Drop files here to upload

![screen shot 2018-10-02 at 5 24 44 pm](https://user-images.githubusercontent.com/191758/46378047-69e03580-c668-11e8-8bd7-7dc045aeae9c.png)

**Comment by shiffman** [comment]
Found another one! I think? The "errors" in the console:

![screen shot 2018-10-02 at 5 34 28 pm](https://user-images.githubusercontent.com/191758/46378415-816bee00-c669-11e8-8532-ddfe6b6af9db.png)

**Comment by catarak** [comment]
thanks for reporting! i need to figure out what these colors should be.

**Comment by Vertmo** [comment]
Moreover, in Dark mode, there doesn't seem to be anything written in the window (on Manjaro Linux, with Firefox):

![capture d ecran_2018-10-03_08-46-08](https://user-images.githubusercontent.com/24356589/46394318-15c96580-c6e9-11e8-88f9-d71dcf164d49.png)

After inspecting the element, it turns out the text is simply white (#fff). It should probably be #333, like in the light mode theme.

**Comment by Rom194** [comment]
And another one : 
![capture d ecran_2018-10-03_19-32-03](https://user-images.githubusercontent.com/25200594/46428014-23133e00-c743-11e8-94ef-52735c62c485.png)
(On ArchLinux with Chrome Version 70.0.3538.35)

**Comment by catarak** [comment]
thanks for reporting more issues!

**Comment by Janglee123** [comment]
and also on login page 
![screenshot from 2018-10-04 00-01-53](https://user-images.githubusercontent.com/29702428/46431193-c88cd880-c768-11e8-90e2-e7b76e968008.png)

**Comment by GaurangTandon** [comment]
Hello @catarak , I'd like to help fix this issue. Is anyone working on this already? Also, what is the minimum accessibility level you'd like to support, AA or AAA?

I plan to fix this by using darker shades of the same colors that were being used, so as to not alter the UI much, but if required, I'll change accordingly. I'll also post screenshots of my changes in my PR.

**Comment by catarak** [comment]
i don't think you'll have to change too many of the colors, and i would try to change as few as possible. the contrast theme was designed to pass the appropriate guideline, however, some color values haven't been set in the right way, and some newer colors got added that don't pass (e.g. in the console).

**Comment by CodingCreate101** [comment]
Is this fixed? I can't reproduce it because I don't have a Mac.

**Comment by CodingCreate101** [comment]
It would be nice to know which are addressed and which aren't. I was going through issues and tried to take the old ones. This one is almost a year old. I'd like to try fixing it. But there was a conversation about design decisions. Not sure if I can actually pick this one.

**Comment by CodingCreate101** [comment]
That's a very good idea. I'll get started with that soon. 🙂

**Comment by catarak** [comment]
Fixed by #1406.
